backend/src/services/notifications.ts

```diff
@@ -0,0 +1,132 @@
+import * as admin from "firebase-admin";
+import schedule from "node-schedule";
+import { Mentee, Mentor, Session } from "../models";
+import { InternalError } from "../errors";
+
+/**
+ * Unicode notes:
+ * eyes - 1F440
+ * cool with glasses - 1F60E
+ * check mark - 2705
+ * broken heart - 1F494
+ *
+ * To find emojis and their unicode value, use https://apps.timwhitlock.info/emoji/tables/unicode
+ * The unicode is in format "U+1234"; in code, use "\u{1234}"
+ */
+
+async function sendNotification(title: string, body: string, deviceToken: string) {
+  const message = {
+    notification: {
+      title,
+      body,
+    },
+    token: deviceToken,
+  };
+
+  admin
+    .messaging()
+    .send(message)
+    .then((response) => {
+      console.log("Notification sent successfully:", response);
+    })
+    .catch((error) => {
+      console.error("Error sending notification:", error);
```

Comment:
This should throw an error rather than logging a message. When calling the function in the route it should catch the error and then print the error statement there.